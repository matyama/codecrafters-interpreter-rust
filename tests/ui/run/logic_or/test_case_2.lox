print 41 or true;
print false or 41;
print false or false or true;

print false or false;
print false or false or false;
print true or true or true or true;
