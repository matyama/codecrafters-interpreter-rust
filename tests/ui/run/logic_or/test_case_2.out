41
41
true
false
false
true
